docs/learn/learn-opengov.md

```diff
@@ -13,323 +13,334 @@ import VLTable from "./../../components/Voluntary-Locking";
+the origin. Root origin has more extended periods than the other origins, such as the Small Tipper
+ones. Also, the number of tracks within each origin differs, with Root origin only accepting one
+proposal at a time (see below).
+
+![opengov-track-capacity](../assets/opengov-track-capacity.png)
+
+This directly affects the number of proposals that can be voted on and executed simultaneously.
+Continuing the comparison between Root and Small Tipper, Small Tipper will allow many proposals on
+different tracks to be executed simultaneously. In contrast, Root will allow only one proposal in
+one track. Once the track capacity is filled, additional proposals in the lead-in period will queue
+until place is available to enter the deciding period.
+
+### Origins and Tracks
+
+An **Origin** is a specific level of privilege that will determine the **Track** of all referenda
+originating from that origin. The track outlines the lifecycle of the proposal and is independent
+from other origins' tracks. The proposer of the referenda now selects an appropriate Origin for
+their request based on the proposal’s requirements.
+
+Although the track structure is the same for all origins, track parameters are not. Such parameters
+include:
+
+- **Maximum Deciding or Capacity**: the limit for the number of referenda that can be decided at
+  once (i.e. the number of tracks within each origin).
+- **Decision deposit**: the amount of funds that must be placed on deposit to enter the Decision
+  Period (note that more requirements must be met to enter the Decision Period).
+- **Preparation Period**: the minimum amount of voting time needed before entering the Decision
+  Period (given capacity and deposit are met).
+- **Decision Period**: the maximum time to approve a proposal. The proposal will be accepted if
+  approved by the end of the period.
+- **Confirmation Period**: the minimum amount of time (within the Decision Period) the approval and
+  support criteria must hold before the proposal is approved and moved to the enactment period.
+- **Minimum Enactment Period**: the minimum amount of waiting time before the proposed changes are
+  applied
+- **Approval Curve**: the curve describing the minimum % of _aye_ votes as a function of time within
+  the Decision Period. The approval % is the portion of _aye_ votes (adjusted for conviction) over
+  the total votes (_aye_, _nay_, and _abstained_).
+- **Support Curve**: the curve describing the minimum % of all votes as a function of time within
+  the Decision Period. The support % is defined as the portion of all votes without conviction (i.e.
+  _aye_, _nay_ and _abstained_) over the total possible amount of votes in the system. Support is a
+  measure of turnout.
+
+So, for example, a runtime upgrade (requiring a `set_code` call, if approved) does not have the same
+implications for the ecosystem as the approval of a treasury tip (`reportAwesome` call), and
```

Comment:
Maybe specify that both use different Origins respectively, unless it is clear enough then disregard.